Dockerfile

```diff
@@ -402,27 +402,25 @@ ARG CGO_CFLAGS="${CFLAGS}"
 ARG CGO_CXXFLAGS="${CXXFLAGS}"
 ARG CGO_LDFLAGS="${LDFLAGS}"
 
-WORKDIR /home/builder/go/src
+WORKDIR /home/builder/sdk-go/src
 RUN ./make.bash --no-clean
 
 # Build the standard library with and without PIE. Target binaries
 # should use PIE, but any host binaries generated during the build
 # might not.
-WORKDIR /home/builder/go
+WORKDIR /home/builder/sdk-go
+ENV GOOS="${GOOS}" \
+  GOROOT="/home/builder/sdk-go" \
+  GOPATH="/home/builder/gopath" \
+  PATH="/home/builder/sdk-go/bin:${PATH}"
 RUN \
-  export GOARCH="${!GOARCH_ARCH}" ; \
```

Comment:
This doesn't look correct to me. IIRC it means the installed standard library will be built for the host and not the target arch, and would need to be rebuilt by each Go build in Bottlerocket proper.